ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -593,8 +625,17 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
   }
 
   ngOnDestroy(): void {
-    this.ngUnsubscribe.next();
-    this.ngUnsubscribe.complete();
+      // finding that some slow http operations
+      // killed if we destroy too early
+      this.componentBusy.pipe(
+        filter(busy => !busy),
+        tap(() => {
+          this.ngUnsubscribe.next();
+          this.ngUnsubscribe.complete();
+          console.debug('unsubscribe completed');
+        }),
+        take(1)
+    ).subscribe();
   }
```

Comment:
Comment describes confronted unsubscribe problem. First time I have noticed this.